apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -87,6 +99,16 @@ export default function sectionAssessments(state=initialState, action) {
 
 // Selector functions
 
+export const getAssessmentList = (state) => {
```

Comment:
nit: since this method returns assessments for the current script, this could be a little clearer as `getCurrentScriptAssessmentList` / `getScriptAssessmentsForCurrentScript` (or something along those lines)

Reply:
& unit test coverage for this method

Reply:
Both great suggestions. I'm going to unblock Nkiru and I pairing for today and merged and do a follow up this afternoon.